### ISSUE TITLE
Syncthing/Filesystem sync [feature request]

### ISSUE THREAD

**Issue by machiav3lli** [PhieF/CarnetDocumentation#26]
It'd be great if it'll be possible to host the files at my own server/folder.
Thanks for the good work.

**Comment by Akito13** [comment]
You can use Nextcloud as your own server. As I neither want to use this program as my server just for Carnet, I thought about enabling basic WebDAV support in this app, since Nextcloud in its essence is based on that, anyway, so it shouldn't be too much work to implement that.

**Comment by PhieF** [comment]
Well it should work very well with external sync apps, have you tried with syncthing ?

**Comment by Amolith** [comment]
@PhieF I just tried syncing the directories over Syncthing but it doesn't seem to be working. `~/QuickNote/` on mobile is being synced to `~/Documents/Carnet/` on my laptop. I just started Carnet on my laptop but the note I created isn't there.

![image](https://user-images.githubusercontent.com/29460675/55267123-c2c70a00-5256-11e9-8751-91b6eb199ce8.png) 
![image](https://user-images.githubusercontent.com/29460675/55267133-cb1f4500-5256-11e9-8350-1d93bf7254af.png)

**Comment by PhieF** [comment]
Did you check the Carnet location on Settings on your desktop ?

Do you see the note in Browser ?

**Comment by Amolith** [comment]
Yes, did you look at the two screenshots I attached? One is showing files in `~/Documents/Carnet` and that the location in Carnet is set to the same directory. The other screenshot is of the browser with no notes in it.
